intranet/apps/eighth/models.py

```diff
@@ -510,7 +519,11 @@ def next_blocks(self, quantity=-1):
         return blocks[:quantity]
 
     def previous_blocks(self, quantity=-1):
-        """Get the previous blocks in order."""
+        """Get the previous blocks in order.
```

Comment:
`Returns` block is improperly formatted and missing return type.  Needs an `Args` block.